test/vm_tests/transformer/test_data/componentWithPartDirective.dart

```diff
@@ -0,0 +1,17 @@
+import 'package:over_react/over_react.dart';
+
+// ignore: uri_does_not_exist, uri_has_not_been_generated
+part 'foo.over_react.g.dart';
+// ignore: uri_does_not_exist, uri_has_not_been_generated
+part 'bar.over_react.g.dart';
```

Comment:
We'll only ever have 1 `over_react.g.dart` part directive in a file, so we should remove lines 5-6